entity-framework/core/get-started/uwp/getting-started.md

```diff
@@ -13,49 +13,42 @@ uid: core/get-started/uwp/getting-started
 
 # Getting Started with EF Core on Universal Windows Platform (UWP) with a New Database
 
-> [!NOTE]  
-> Temporarily this tutorial uses EF Core 1.1. UWP has not been updated yet to support .NET Standard 2.0 which is required for compatibility with EF Core 2.0. Once it is, we will update the tutorial to use the new version. 
+-> [!NOTE]
+-> This tutorial uses EF Core 2.0.1. EF Core 2.0.0 lacks some crucial bugfixes required for a good UWP experience.
 
 In this walkthrough, you will build a Universal Windows Platform (UWP) application that performs basic data access against a local SQLite database using Entity Framework.
 
-> [!WARNING]  
-> **Avoid using anonymous types in LINQ queries on UWP**. Deploying a UWP application to the app store requires your application to be compiled with .NET Native. Queries with anonymous types have poor performance on .NET Native or may crash the application.
+> [!IMPORTANT]
+> **Consider avoiding anonymous types in LINQ queries on UWP**. Deploying a UWP application to the app store requires your application to be compiled with .NET Native. Queries with anonymous types have worse performance on .NET Native.
 
-> [!TIP]  
+> [!TIP]
 > You can view this article's [sample](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/GetStarted/UWP/UWP.SQLite) on GitHub.
 
 ## Prerequisites
 
 The following items are required to complete this walkthrough:
-* Windows 10
 
-* [Visual Studio 2017](https://www.visualstudio.com/downloads/)
+* [Windows 10 Fall Creators Update](https://support.microsoft.com/en-us/help/4027667/windows-update-windows-10) (10.0.16299.0)
+
+* [.NET Core 2.0.0 SDK](https://www.microsoft.com/net/core) or later.
+
+* [Visual Studio 2017](https://www.visualstudio.com/downloads/) version 15.4 or later with the **Universal Windows Platform Development** workload.
 
-* The latest version of [Windows 10 Developer Tools](https://dev.windows.com/downloads)
+## Create a new model project
 
-## Create a new project
+> [!WARNING]
+> Due to a limitation in the Entity Framework tools the model needs to be placed in a non-UWP project to be able to run migrations commands in the Package Manager Console
```

Comment:
I don't think we should make this sound like it is just EF tools that are limited. Maybe, "Due to limitations in the way .NET Core tools interact with UWP..."